ready_recipes/arm_pyart/build.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+if [ `uname` == Darwin ]; then
+    # Remove gfortran so that all fortran Py-ART modules are not built.  This
+    # avoids the need to package the gfortran run time in the conda package.
```

Comment:
> Longer term the plan is to re-write the Fortran module in Cython.

Longer term plan for conda-forge will be to make this easier somehow. Whether that is packaging libgfortran, or some other solution. I'm certainly keen to avoid the need to shy away from using Fortran.